### PR TITLE
add(): modal

### PR DESCRIPTION
**Pull request by dan-entous**
## purpose
prepare for the new `EloquaForms` component so that it can be placed into a modal. add the new `Modal` component and move the current `ModalButton` component into the same folder.

## 🤾‍♂ playground
* click on the download cura link to make sure it's still working:
    * https://playground-modal-old.ultimaker.k8s-dev.ultimaker.works/software/ultimaker-cura
    * https://playground-modal-new.ultimaker.k8s-dev.ultimaker.works/software/ultimaker-cura
* click on the watch video link in the hero to make sure it's still working:
    * https://playground-modal-old.ultimaker.k8s-dev.ultimaker.works/3d-printers
    * https://playground-modal-new.ultimaker.k8s-dev.ultimaker.works/3d-printers

### CODE REVIEW

File: src/atoms/cta/ModalButton.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { ModalWizard } from './ModalWizard';
 import { AnalyticalEvent } from '../../events/AnalyticalEvent';
+import { Modal } from '../../molecules/modal/Modal';
 
 export class ModalButton {
     clickEvent?: AnalyticalEvent;
     label: string;
-    modal: ModalWizard;
+    modal: Modal;
```
Comment (evdtempel) [inline src/atoms/cta/ModalButton.ts]: Both ModalWizard and Wizard should be present here for backwards-compatibility.

File: src/molecules/wizard/Wizard.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { WizardStepInterface } from './WizardStepInterface';
 
-export class ModalWizard {
+export class Wizard {
```
Comment (evdtempel) [inline src/molecules/wizard/Wizard.ts]: Both `ModalWizard` and `Wizard` should exist for backwards-compatibility.

File: src/atoms/cta/ModalButton.ts (hunk 1 above)
Comment (dan-entous) [reply src/atoms/cta/ModalButton.ts]: i thought so as well, but after further investigating it, maintaining them wasn’t necessary. we can go over it together once i’m in the office.

File: src/molecules/wizard/Wizard.ts (hunk 2 above)
Comment (dan-entous) [reply src/molecules/wizard/Wizard.ts]: i thought so as well, but after further investigating it, maintaining them wasn’t necessary. we can go over it together once i’m in the office.

**Pull request merged**